server/services/mi-light/lib/light/light.configureBridge.js

```diff
@@ -0,0 +1,66 @@
+const logger = require('../../../../utils/logger');
+const { NotFoundError } = require('../../../../utils/coreErrors');
+const {
+  BRIDGE_EXTERNAL_ID_BASE,
+  BRIDGE_MODEL,
+  BRIDGE_IP_ADDRESS,
+  BRIDGE_MAC,
+  BRIDGE_NAME,
+  BRIDGE_TYPE,
+} = require('../utils/consts');
+
+/**
+ * @description Configure the milight bridge.
+ * @param {string} Mac - Mac Address of the Mi-Light Bridge.
+ * @example
+ * configureBridge('F0:FE:6B:A6:03:EB');
+ */
+async function configureBridge(Mac) {
+  const bridge = this.bridgesByMac.get(Mac);
+  if (!bridge) {
+    throw new NotFoundError(`BRIDGE_NOT_FOUND`);
+  }
+  logger.info(`Connecting to milight bridge "${Mac}", ip = ${bridge.ip}, name = ${bridge.name}`);
+  try {
+    const bridgeMacWithoutColomn = Mac.replace(/:/g, '');
+    const deviceCreated = await this.gladys.device.create({
```

Comment:
Can't you create device from front ? Calling the create devie API.

Reply:
I used the phillips hue as example. Pierre-Gilles did it like this.

Reply:
This function is only called from the front, so it's the same thing :) 

The thing I want to avoid, is to have "background creation of device" => we don't want that, because it can results in hard to understand errors.

Here, this route is called by the front, so if it fails, we can display an error message, so it's ok.